packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -231,17 +231,6 @@ class AvatarGroup extends ResizeMixin(OverlayClassMixin(ElementMixin(ThemableMix
         value: () => [],
       },
 
-      /** @private */
-      __maxReached: {
```

Comment:
This property was only used in one observer so I decided to inline it there, so that we only have one observer for updating overflow avatar instead of two different ones.